File: java/oop/AccountPolymorphismApp/src/com/techlabs/account/SavingAccount.java
package com.techlabs.account;

public class SavingAccount extends Account {
	public SavingAccount(int acno,String  acntname,double balance) {
		super(acno,acntname,balance);
	}
	public void withdraw(double amt){
		if(this.balance-amt<=500){
			System.out.println("Cannot withdraw....Limit Exceed");
		}else{
			this.balance=this.balance-amt;
		}
	}

}

File: dotNet/csharp/AccountPolymorphismApp/AccountPolymorphismApp/SavingAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AccountPolymorphismApp
{
    class SavingAccount : Account
    {
        public SavingAccount(int acno, string acntname, double balance) : base(acno, acntname, balance)
        {
        }

        public override void Withdraw(double amt)
        {
            if (this._balance - amt <= 500)
            {
                Console.WriteLine("Cannot withdraw....Limit Exceed");
            }
            else
            {
                this._balance = this._balance - amt;
            }
        }
    }
}
